tests/clients/test_device_settings.py

```diff
@@ -827,3 +836,45 @@ def test_backup_set_filename_exclusions_append_produces_expected_values(
             {"@regex": PICTURES_REGEX},
             {"@regex": NEW_REGEX},
         ]
+
+    def test_backup_set_when_locked_returns_expected_property_value(
+        self, device_settings_with_locked_backup_set
+    ):
+        device_settings = DeviceSettings(device_settings_with_locked_backup_set)
+        assert device_settings.backup_sets[0].locked
+
+    def test_backup_set_when_set_locked_non_destination_attributes_raise_attr_error_when_set(
+        self, device_settings_with_locked_backup_set
+    ):
+        device_settings = DeviceSettings(device_settings_with_locked_backup_set)
+        with pytest.raises(AttributeError):
+            device_settings.backup_sets[0].included_files.append("test")
+
+        with pytest.raises(AttributeError):
+            device_settings.backup_sets[0].included_files = ["test"]
+
+        with pytest.raises(AttributeError):
+            device_settings.backup_sets[0].excluded_files.append("test")
+
+        with pytest.raises(AttributeError):
+            device_settings.backup_sets[0].excluded_files = ["test"]
+
+        with pytest.raises(AttributeError):
+            device_settings.backup_sets[0].filename_exclusions.append("test")
+
+        with pytest.raises(AttributeError):
+            device_settings.backup_sets[0].filename_exclusions = ["test"]
+
+    def test_backup_set_when_set_locked_allows_destination_modifications(
```

Comment:
Curious if this is supposed to be `test_backup_set_when_not_locked_allows_destination_modifications()`
(second `set` -> `not`).

Reply:
No, this is correct, because destination usages _are_ still modifiable when the whole backup set is locked.